test/integration/data_bags/osl-app/formsender.json

```diff
@@ -0,0 +1,4 @@
+{
+  "token": "15674hsda//*q23%^13jnxccv3ds54sa4g4sa532323!OoRdsfISDIdks38*(dsfjk)aS",
+  "recaptcha_secret": "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

Comment:
I hope this isn't real tokens??

Reply:
These are the values in formsender's current `conf.py.dist`: https://github.com/osuosl/formsender/blob/master/conf.py.dist

Let me know if/where you want me to move them though

Reply:
As I mentioned on Slack. please come up with different values for these here.